app/views/govuk_wrapper.scala.html

```diff
@@ -117,7 +117,7 @@
 }
 
 @mainContentHeader = {
-      <div class="float--right">@views.html.language_selection(
+      <div class="text--right">@views.html.language_selection(
```

Comment:
There's quite a lot of gap between the language toggle and the back button - one way you could remove this is removing the `<div>` that surrounds the language_selection() helper and add a custom class as parameter to the `language_selection()` call for example:
```
    @if(appConfig.features.useLanguageSelector()){
        @views.html.language_selection(
            appConfig.languageMap,
            appConfig.routeToSwitchLanguage,
            customClass = Some("text--right")
        )
    }
```